code/game/machinery/doors/airlock.dm

```diff
@@ -767,9 +768,10 @@ About the new airlock wires panel:
 			set_airlock_overlays(AIRLOCK_DENY)
 			if(density && arePowerSystemsOn())
 				flick("deny", src)
+				playsound(loc, open_failure_access_denied, 50, 0)
 				if(secured_wires && world.time > next_clicksound)
 					next_clicksound = world.time + CLICKSOUND_INTERVAL
-					playsound(loc, open_failure_access_denied, 50, 0)
+					playsound(loc, open_failure_access_denied_high_sec, 50, 0)
```

Comment:
This is going to play both sounds at once if it is a high-security door.

Reply:
I didnt notice this during testing due to the high security sound overlapping it, Im not opposed to just yeeting the high access sound and making a uniform door sound